.github/environment.yml

```diff
@@ -0,0 +1,12 @@
+name: zchunk_test_env
+channels:
+- conda-forge
+dependencies:
+- libcurl
+- zstd
+- meson
+- c-compiler
+- ninja
+- pkg-config
+- cmake
+- sel(osx): argp-standalone
```

Comment:
Can we please get `openssl-devel` (or whatever its equivalent is)?

Reply:
In the packages created by `conda-forge` the headers are always included so no `...-devel` packages are needed.

Reply:
sorry, missed that! Done.